java/google/registry/rde/imports/RdeHostLinkAction.java

```diff
@@ -117,10 +117,18 @@ public void map(JaxbFragment<XjcRdeHostElement> fragment) {
         ofy().transact(new VoidWork() {
           @Override
           public void vrun() {
+            // link host to superordinate domain and set time of last superordinate change to
+            // the time of the import
             HostResource host =
                 ofy().load().now(Key.create(HostResource.class, xjcHost.getRoid()));
             ofy().save()
-                .entity(host.asBuilder().setSuperordinateDomain(superordinateDomainKey).build());
+                .entity(host.asBuilder().setSuperordinateDomain(superordinateDomainKey)
+                    .setLastSuperordinateChange(ofy().getTransactionTime())
+                    .build());
+            // link domain to subordinate host
+            DomainResource domain = ofy().load().now(superordinateDomainKey);
```

Comment:
Add a comment that you're intentionally reloading the domain inside the transaction for a reason (in case it was changed by something else in the mean time), otherwise people might wonder why you aren't just using the superordinateDomain variable from above.